akka-ssl-config/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package com.typesafe.sslconfig.akka
```

Comment:
So the general idea was to apply to akka-specific types things in this lib, while ssl-config-core only can touch SSLContext / Params etc. AFAICS current state of lib does not really need akka specific types other than setting up the logger etc. This extension allows changing the settings before being passed into akka http programatically, in case you have more apps in the same jvm etc.

I agree with changing the package name here to perhaps com.lightbend